orchagent/aclorch.cpp

```diff
@@ -1178,6 +1172,12 @@ bool AclRuleMirror::create()
     sai_object_id_t oid = SAI_NULL_OBJECT_ID;
     bool state = false;
 
+    if (!m_pMirrorOrch->sessionExists(m_sessionName))
```

Comment:
Moving mirror session existence check here is to ensure acl rule will be retried if mirror session is absent instead of being erased (`bAllAttributesOk` == false logic)?

Reply:
That's right!